### DISCUSSION TITLE
Need some clarification

### DISCUSSION THREAD

**Discussion by muhshif** [btschwertfeger/python-cmethods#26] [Q&A]
Clarify me if my understanding which I am writing below about the 3 datasets you mentioned in CMethods is wrong:

Observation data: means the real time observation data with which we compare our model simulation data.
Control data: The historical simulation data or hindcast data.
Scenario data: The future projection data or forecast data.

Correct me if my understanding is wrong.

**Comment by btschwertfeger** [comment]
That's correct. The name "obs" for observation/reference data was not a well-chosen one on my part. It would have been better to use "reference" instead. Assuming I have modeled (A) and observed (B) daily precipitation data for the years 1991-2020 (control period) and want to adjust modeled data simulating the years 2001-2030 (C), then A is the "obs" (control period), B is the "simh" (control period), and C is the "scen" (scenario period).
